code/frameworks/angular/src/server/angular-cli-webpack-13.x.x.ts

```diff
@@ -45,7 +40,9 @@ exports.getWebpackConfig = async (baseConfig, { builderOptions, builderContext }
     (wco) => [
       getCommonConfig(wco),
       getStylesConfig(wco),
-      getTypeScriptConfig ? getTypeScriptConfig(wco) : getDevServerConfig(wco),
+      // todo getTypeScriptConfig is not exported and probably always used getDevServerConfig. Check what to do
+      // Previous code `getTypeScriptConfig ? getTypeScriptConfig(wco) : getDevServerConfig(wco),`
+      getDevServerConfig(wco),
```

Comment:
I don't know if this is useful or not and in what circumstances :

13.0.0 :
```
      getDevServerConfig(wco),
      getCommonConfig(wco),
      getBrowserConfig(wco),
      getStylesConfig(wco),
      getStatsConfig(wco),
      getAnalyticsConfig(wco, context),
      getTypeScriptConfig(wco),
      wco.buildOptions.webWorkerTsConfig ? getWorkerConfig(wco) : {},
```
13.1.0 : 
```
        getDevServerConfig(wco),
        getCommonConfig(wco),
        getStylesConfig(wco),
        getAnalyticsConfig(wco, context),
```

15.0.0 : 
```
        getDevServerConfig(wco),
        getCommonConfig(wco),
        getStylesConfig(wco)
```

`getDevServerConfig` is only used by : `builders/dev-server` not by `/builders/browser` 🤷‍♂️  
I don't think stroybook needs it. but maybe he should 

And for `getTypeScriptConfig` in 13.0.0 same 🤣